use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResource.java

```diff
@@ -179,6 +183,21 @@ public VehicleRoutePlan terminateSolving(
         return getRoutePlan(jobId);
     }
 
+    @Operation(summary = "Submit a route plan to analyze its score.")
+    @APIResponses(value = {
+            @APIResponse(responseCode = "202",
```

Comment:
This is a synchronous call that returns immediately; wouldn't be returning 200 more appropriate?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202

Reply:
Good catch!

Reply:
Done.